packages/discovery-react-components/src/components/CIDocument/__stories__/default.md

```diff
@@ -0,0 +1,3 @@
+#### Overview
+
+The `CIDocument` component provides Element Classification to the user. It takes a PDF and enrich it with features a customer might deem important (such as obligation, Deliverables, Assignments, ect...). This would help a customer get a better understanding of their document or help them find what they need in a large document in a more efficient way.
```

Comment:
A few of grammar nitpicks.

The O in obligation should be capitalized to match the other examples.
Change ect to etc.
Change `This would help a customer` to `This would help customers`
Enrich should probably be enriches.  
Also in the third sentence, there should be a comma after the word `document`.

Other than those minor things, looks good.

Reply:
Done ✅